scripts/docs-collator/templates/github-actions/readme.md

```diff
@@ -0,0 +1,81 @@
+{{- defineDatasource "config" .Env.README_YAML | regexp.Replace ".*" "" -}}
+{{- defineDatasource "includes" .Env.README_INCLUDES | regexp.Replace ".*" "" -}}
+{{- $deprecated := has (ds "config") "deprecated" -}}
+{{- $fullName := (ds "config").name -}}
+{{- $shortName := (index ($fullName | strings.SplitN "-" 3) 2) -}}
+---
+title: {{ $shortName }}
+sidebar_label: {{ $shortName }}
+sidebar_class_name: command
+{{- if has (ds "config") "description" }}
+description: |-
+{{ (ds "config").description | strings.Indent 2 }}
+{{- end }}
+{{- if has (ds "config") "tags" }}
+tags:
+{{ (ds "config").tags | data.ToYAML | strings.Indent 2 -}}
+{{- end }}
+custom_edit_url: https://github.com/cloudposse/{{ $fullName }}/edit/master/README.md
+---
+
+# GitHub Action: `{{ $shortName }}`
+
+{{- if $deprecated }}
+## Deprecated
+
+{{ if has (ds "config").deprecated "notice" }}
+  {{- (ds "config").deprecated.notice }}
+{{- else }}
+  This module is no longer actively maintained
+{{- end }}
+
+{{ if has (ds "config") "description" }}
+### Historical Description
```

Comment:
Why is it the "Historical" description?

Reply:
I took the template from here https://github.com/cloudposse/build-harness/blob/master/templates/README.md.gotmpl#L60 and simplified it.